README.md

```diff
@@ -33,19 +35,393 @@ yarn chain
 > in a second terminal window, 🛰 deploy your contract (locally):
 
 ```sh
-cd {challengeName}
+cd challenge-4-dex
 yarn deploy
 ```
 
 > in a third terminal window, start your 📱 frontend:
 
 ```sh
-cd {challengeName}
+cd challenge-4-dex
 yarn start
 ```
 
 📱 Open http://localhost:3000 to see the app.
 
-## Checkpoint {num}: Create all the required Checkpoints for the challenge
+## ⛳️ Checkpoint 1: 🔭 The Structure 📺
+Navigate to the Debug Contracts tab and you should see two smart contracts displayed called DEX and Balloons.
+
+👩‍💻 Rerun yarn deploy whenever you want to deploy new contracts to the frontend (run yarn deploy --reset for a completely fresh deploy if you have made no contract changes).
+
+Balloons.sol is just an example ERC20 contract that mints 1000 $BAL to whatever address deploys it. DEX.sol is what we will build in this challenge and you can see it starts with a SafeMath library to help us prevent overflows and underflows and also tracks a token (ERC20 interface) that we set in the constructor (on deploy).
```

Comment:
backticks for balloons.sol and dex.sol. Also probably we need to name it `packages/hardhat/contracts/Baloons.sol` like it's in other challenges